app/client/api/userAPI.js

```diff
@@ -54,4 +54,22 @@ export const editUserAPI = ({body, path}) => {
     .catch(error => {
         throw error;
     })
-};
\ No newline at end of file
+};
+
+export const resetPinAPI = (userId) => {
```

Comment:
we gotta refactor some of this. tons of code repeat across the APIs

Reply:
added todo: https://trello.com/c/AJBoE83s/136-refactor-web-api